tests/integration/API/Catalog/Send_Item_Updates/RequestTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Tests\API\Catalog\Send_Item_Updates;
+
+use SkyVerge\WooCommerce\Facebook\API\Catalog\Send_Item_Updates\Request;
+
+/**
+ * Tests the API\Catalog\Send_Item_Updates\Request class.
+ */
+class RequestTest extends \Codeception\TestCase\WPTestCase {
+
+
+	/** @var \IntegrationTester */
+	protected $tester;
+
+
+	public function _before() {
+
+		parent::_before();
+
+		require_once 'includes/API/Request.php';
+		require_once 'includes/API/Catalog/Send_Item_Updates/Request.php';
+	}
+
+
+	/** Test methods **************************************************************************************************/
+
+
+	/** @see Request::__construct() */
+	public function test_constructor() {
+
+		$request = new Request( '1234' );
+
+		$this->assertEquals( '/1234/batch', $request->get_path() );
+		$this->assertEquals( 'POST', $request->get_method() );
+	}
+
+
+	/** @see Request::set_requests() */
+	public function test_set_requests() {
+
+		$request = new Request( '1234 ');
+
+		$request->set_requests( [] );
+
+		// TODO: assert that the array of requests is included in get_data()
```

Comment:
What is missing so that this can be added?

Reply:
We have another [story](https://app.clubhouse.io/skyverge/story/54734/prepare-the-data-for-the-send-item-updates-request) to overwrite the `get_data()` method in this class and return a value that would include the values from the requests and the allow user properties.

I thought I could use get_data() for testing instead of accessing the protected property with reflection.